bundle.Dockerfile

```diff
@@ -1,17 +1,13 @@
 FROM scratch
 
-LABEL com.redhat.openshift.versions="v4.8"
-LABEL com.redhat.delivery.backport=true
-LABEL com.redhat.delivery.operator.bundle=true
```

Comment:
Not needed any more?

Reply:
Looks like these removed by the new controller-runtime version. Anyway, the bundle is built by the RedHat. Any labels from the `bundle.Dockerfile` are replaced by the values from the `annotations.yaml`